sourced/ml/cmd/bigartm2asdf.py

```diff
@@ -39,4 +39,4 @@ def bigartm2asdf(args):
     indices = numpy.array(indices, dtype=numpy.int32)
     matrix = csr_matrix((data, indices, indptr), shape=(len(tokens), len(items) - 2)).T
     Topics().construct(tokens=tokens, topics=None, matrix=matrix) \
-        .save(output=args.output, series="topics")
+        .save(output=args.output, series="bigartm")
```

Comment:
so the `series` should correspond to the problem the model is solving. In our case, this is topic modeling, so we should leave "topics"